parsons/etl/table.py

```diff
@@ -50,6 +51,10 @@ def __init__(
     ):
         self.table = None
 
+        #
+        if inspect.isgenerator(lst):
```

Comment:
it's very convenient to write pagination code as a generator as i've done in this PR, and imo parsons should support that. https://github.com/move-coop/parsons/issues/1529